Godeps/_workspace/src/k8s.io/kubernetes/pkg/client/unversioned/debugging.go

```diff
@@ -135,6 +135,14 @@ func (rt *DebuggingRoundTripper) RoundTrip(req *http.Request) (*http.Response, e
 	return response, err
 }
 
+func (rt *DebuggingRoundTripper) CancelRequest(req *http.Request) {
+	if canceler, ok := rt.delegatedRoundTripper.(requestCanceler); ok {
+		canceler.CancelRequest(req)
```

Comment:
This is following the existing code, is an upstream PR, and should show up in the next rebase.